app/controllers/profiles_controller.rb

```diff
@@ -10,7 +10,7 @@ def edit
 
   def update
     if current_user.update(user_params)
-      redirect_to edit_profile_path, flash: { warning: 'User was successfully updated.' }
+      redirect_to profile_path, flash: { warning: 'User was successfully updated.' }
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.